### DISCUSSION TITLE
Impossible to filter Pull Request on Authors who aren't collaborators

### DISCUSSION THREAD

**Discussion by jjerphan** [community/community#40690] [Mobile]
Hi!

First, thank you for your work on developing GitHub on mobile ! 

On GitHub Android App (version 1.91.0) it seems impossible to filter Pull Requests on Authors who aren't collaborators: only the collaborators of the repository show up as proposed and no other users is found even if their full GitHub handle is entered.

Is it a problem on my end or some room for improvement?

Julien.

**Comment by jfullstackdev** [comment]
workaround, you can log in using your browser (preferably Chrome) and all of the functionalities of GitHub Web should work

**Reply by jjerphan** [reply]
Yes, this is what I do currently.
